internal/services/groups_test.go

```diff
@@ -5,16 +5,23 @@ import (
 	"testing"
 
 	featureFlagsConstants "github.com/checkmarx/ast-cli/internal/constants/feature-flags"
+
 	"github.com/checkmarx/ast-cli/internal/wrappers"
 	"github.com/checkmarx/ast-cli/internal/wrappers/mock"
 )
 
+func setup() {
```

Comment:
The issue is that if the map is not restarted, its data persists from the previous test, and updating the mock doesn't help because the new logic is that if an item is found in the map (cache), no HTTP request is made. Therefore, updating the mock doesn't make a difference.